sql/core/src/test/resources/sql-tests/inputs/date.sql

```diff
@@ -91,6 +91,11 @@ create temp view v as select '1' str;
 select date_add('2011-11-11', str) from v;
 select date_sub('2011-11-11', str) from v;
 
+-- non-literal string column add/sub with integer
+create or replace temp view v2 as select '2011-11-11' str;
```

Comment:
nit: let's remove `or replace`. It's always error-prone if we replace a temp view in the test, as we may change the rest of the test cases.